### ISSUE TITLE
Data Model thinking...

### ISSUE THREAD

**Issue by robzolkos** [robzolkos/turbonativedirectory#3]
Model:

App/Application ?  (feels weird to have a model called this _in_ an app)

Attributes:

- name
- type  ios/android/both?
- url -> to app store

has_many :screenshots

Screenshot
  has_attached :file

**Comment by MikeRayUX** [comment]
I'm thinking `App` is a fitting choice here, as it inherently refers to applications deployed in stores.

`platform` might also be a better fit than `type` (unsure whether `type` is still a reserved keyword for STI).

Screenshots:

Would it be helpful to display screenshots/descriptions on a show page/modal?

**Comment by MikeRayUX** [comment]
For Apps that are in both stores:

`app_store_url`, `play_store_url` 

OR  

`ios_url`, `android_url` would probably make more sense than an enum.

**Comment by robzolkos** [comment]
Yeah the platform enum is great.  Each app will have its own show page for sure (probably shown in a modal).

**Comment by williamkennedy** [comment]
Hey @robzolkos 

I had a [tweet](https://twitter.com/_williamkennedy/status/1706583157540065620) about this the other day, but you beat me to it. My tweet was only half a joke 😁

But something that I do think it is worth pursuing is just using a yaml file for your data. This means that you can load it from a file and don't have to worry about a database. It will help save some time and money.

The benefit of a yaml file is that it would make it easier for contributors to add another record compared with a database interface. 

It's just a thought.

**Comment by robzolkos** [comment]
Hey @williamkennedy - yep I just added a config/app_directory.yml file!   Much easier!
